apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -800,6 +1058,112 @@ SOMAExperimentAxisQuery <- R6::R6Class(
       }
       return(mat)
     },
+    .load_df = function(df = 'obs', index = NULL, columns = NULL) {
```

Comment:
`df` -> `df_name` -- `df` makes it sound like it _is_ a dataframe (it isn't) leading to a little unnecessary cognitive hiccup

Reply:
Also, passing along a pro-tip from @eddelbuettel you can have `df_name = c('obs', 'var')` -- and  then `df_name <- match.arg(df_name)` -- for added clarity to code-users as well as code-readers/code-maintainers

Reply:
likewise `columns` sounds like they are actual columns, leading to another cognitive hiccup when it turns out to be not the case: please `column_names`